#### Домашняя работа №5

##### Задание №1

1. Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
from math import sqrt

Z_1 = 1.96

In [2]:
std = 16
a = 0.95
M = 80
n = 256

T1 = M - Z_1 * std / sqrt(n) 
T2 = M + Z_1 * std / sqrt(n)

print(T1, T2)

78.04 81.96


Ответ: доверительный интервал для оценки математического ожидания от 78.04 до 81.96

#### Задание №2

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [3]:
n = 10
X = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
avg_x = sum(X) / len(X)
std_x = sqrt(sum(map(lambda x: (x - avg_x)**2, X)) / len(X))
Z = 1.812 

T3 = avg_x - Z * std_x / sqrt(n) 
T4 = avg_x + Z * std_x / sqrt(n)

T = (T3 + T4) / 2
T

6.589999999999999

Ответ: истинное значение величины X составляет 6.589999999999999

##### Задание №3

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

In [4]:
X_hyp = 17
a = 0.05
n = 100
D_mean = 17.5
D = 4
std_v = sqrt(D)
Z_a = 1.645

Z_hyp = (D_mean - X_hyp) / (std_v / sqrt(n))
Z_hyp

# Z_hyp < Z_a - гипотеза не подтверждается

2.5

Ответ: поскольку Z_hyp больше Z_a гипотеза не подтверждается.

##### Задание №4

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

- не успеваю :(